src/nix/main.cc

```diff
@@ -59,7 +59,6 @@ struct HelpRequested { };
 
 struct NixArgs : virtual MultiCommand, virtual MixCommonArgs
 {
-    bool printBuildLogs = false;
```

Comment:
Looks like it was removed, not renamed? :P

Reply:
Oops yes I typed the wrong thing.